### PR TITLE
Replace String#substring() with String#startsWith() or String#endsWith()

### PR DESCRIPTION
**Pull request by segayuu**
And String#indexOf() with String#includes()

- [ ] Add test cases for the changes.
- [ ] Passed the CI test.

### CODE REVIEW

**Comment by segayuu** [comment]
Since I have not responded for more than a week since I requested a review, I will notify @hexojs/core member once more.
If there is no response after one week from this comment, we will merge.

**Comment by ivan-nginx** [comment]
@segayuu > If there is no response after one week

Why week? Go about 1-2 month? It's normal period for awaiting something important.

Also, i see codeclimate issues, mb u can fix them?

File: lib/plugins/filter/meta_generator.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
```
Comment (ivan-nginx) [inline lib/plugins/filter/meta_generator.js]: Also, it seems new filter added here. But in pull readme there is nothing about this.

Comment (segayuu) [reply lib/plugins/filter/meta_generator.js]: That change is the result of # 3129 being resolved by running git rebase.

**Pull request merged**